modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/macros/ProcessBuilderKaleoDesignerReact.macro

```diff
@@ -764,6 +775,17 @@ definition {
 			locator1 = "ProcessBuilderKaleoDesignerReact#SELECTED_NODE");
 	}
 
```

Comment:
You could create a macro to assert that the translation language it's present, to use before you delete the chosen one.